lib/modulr/resources/payments/payment.rb

```diff
@@ -5,8 +5,11 @@ module Resources
     module Payments
       class Payment < Base
         map :id, [:id, :payment_reference_id]
-        map :status, :status
+        map :status
+        map :reference
         map :externalReference, :external_reference
+        map :createdDate, :created_at
+        map :approvalStatus, :approval_status
```

Comment:
We need to include the `details` of payment, but just wanted to release this version now. So I'm leaving the addition of that info for the next PR.

Reply:
Yes, I need this in order to [finish this PR](app/domains/banking/services/providers/modulr/transaction_finder.rb).

```
NoMethodError:
       undefined method `details' for #<Modulr::Resources::Payments::Payment:0x00007fa95b04bad0>
```

Reply:
@IvanGuardado [4db7ee5](https://github.com/devengoapp/modulr-ruby/pull/8/commits/4db7ee570fa053499a91d0a463e05c878c11d96f) introduces it.